qadence2_platforms/backends/analog/functions.py

```diff
@@ -67,6 +68,45 @@ def dyn_pulse(
     sequence.add(p, channel="global")
 
 
+def piecewise_pulse(
+    sequence: Sequence,
+    duration: Variable | VariableItem,
+    amplitude: Variable,
+    detuning: Variable,
+    phase: VariableItem | float,
+    **_: Any,
+) -> None:
+    """
+    Dynamic pulse to simulate a specific piecewise time-dependent hamiltonian for neutral-atom devices.
+
+    :param sequence: a `pulser.sequence.Sequence` instance
+    :param duration: duration of the pulse in nanoseconds
+    :param amplitude: amplitude of the pulse in rad/µs
+    :param detuning: detuning of the pulse in rad/s
+    :param phase: phase in rad
+    """
```

Comment:
We use google's docstring style. I guess this docstring will now not be picked up by MKdocs.